tldr.cabal

```diff
@@ -53,6 +54,7 @@ library
   ghc-options: -Wall -O2
   build-depends:
       ansi-terminal
+    , attoparsec
```

Comment:
Can you do the changes in `package.yaml` as cabal file is generated from there ?